### PR TITLE
Hugo documentation with Docsy theme

### PR DESCRIPTION
**Pull request by Sam032**

### CODE REVIEW

File: resources/_gen/assets/js/js/search.js_b4bb3e7b21b43e920b7c367ff787d383.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"Target":"js/search.js","MediaType":"application/javascript","Data":{}}
```
Comment (nissessenap) [inline resources/_gen/assets/js/js/search.js_b4bb3e7b21b43e920b7c367ff787d383.json]: Should the resources folder be in the git repo?

File: static/CNAME [hunk 2]
```diff
@@ -0,0 +1 @@
+docs.nosy.tech
```
Comment (nissessenap) [inline static/CNAME]: I don't think this is needed since we won't host the site in this repo.

File: .yaspellerrc [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    "NoSy",
+    "img",
+    "Keycloak",
+    "IAM",
+    "openid",
+    "zookeeper",
+    "CC",
+    "base64",
+    "linkTitle",
+    "npm",
```
Comment (nissessenap) [inline .yaspellerrc:14]: Don't we need yaspellerrc in the dictory of ignores?

**Review by nissessenap — CHANGES_REQUESTED** [review]
Please have a look at comments

File: .yaspellerrc (hunk 3 above)
Comment (Sam032) [reply .yaspellerrc:14]: Added yaspellerrc to ignore list for spellcheck

File: static/CNAME (hunk 2 above)
Comment (Sam032) [reply static/CNAME]: Removed CNAME file

**Review by nissessenap — APPROVED** [review]

**Pull request merged**